plugins/callback/foreman.py

```diff
@@ -249,8 +262,11 @@ def send_reports(self, stats):
                     },
                     "logs": list(build_log(self.items[host])),
                     "reporter": "ansible",
+                    "check_mode": self.check_mode,
                 }
             }
+            if self.check_mode == '1':
+                report['config_report']['status']['pending'] = total['ok']
```

Comment:
"ok" sounds like the ones that are unchanged? and you want "changed"?

Reply:
@evgeni Thanks I did not know this information already exists, I will use this.
regarding the changed vs ok I think that is what we discussed in the ansible meeting. @ares @ezr-ondrej do you think the pending should be the changed tasks or all the ok tasks?

Reply:
Pending should be the tasks that would have been changed without check mode I think.

Reply:
yeah, `changed` should mean pending, unchanged, should probably stay untouched, as those are not pending changes.

Reply:
Ok, I will fix this